formatTest/unit_tests/input/jsx.re

```diff
@@ -32,15 +32,15 @@ let y =
 let z =
   <div
     style=ReactDOMRe.Style.make(
-              ::width,
-              ::height,
-              ::color,
-              ::backgroundColor,
-              ::margin,
-              ::padding,
-              ::border,
-              ::borderColor,
-              ::someOtherAttribute,
+              :width,
+              :height,
+              :color,
+              :backgroundColor,
+              :margin,
+              :padding,
+              :border,
+              :borderColor,
+              :someOtherAttribute,
```

Comment:
How come it's indented like this?

Reply:
Because it's the *input* of the test & we make sure the indentation is correct in the * expected output ⛷  😄

Reply:
Oops...